Engine/Component/ComponentCamera.h

```diff
@@ -49,7 +50,7 @@ class ComponentCamera : public Component
 	float GetHeight() const;
 
 	void RecordFrame(float width, float height);
-	void RecordDebugDraws(float width, float height) const;
+	void RecordDebugDraws(float width, float height);
```

Comment:
Same as before.

Reply:
It's the same yep, in order to pass the camera `this` as a parameter within the method, const needed to begone